spec/vsphere_connection_handling_spec.rb

```diff
@@ -25,7 +25,7 @@
 
     context 'password is not in config file' do
       let(:password) { nil }
-      let(:ui) { double( 'Ui', ask: 'passwurd') }
+      let(:ui) { double('Ui', ask: 'passwurd') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.